File: plugin/src/main/java/me/aurealisch/pudgeland/Pudgeland.java
package me.aurealisch.pudgeland;

import me.aurealisch.pudgeland.commandExecutors.ConnectCommandExecutor;
import me.aurealisch.pudgeland.commandExecutors.DisconnectCommandExecutor;
import org.bukkit.plugin.java.JavaPlugin;

import java.util.Objects;
import java.util.logging.Logger;

@SuppressWarnings("unused")
public final class Pudgeland extends JavaPlugin {
    Logger logger;
    WebSocketClientEndpoint webSocketClientEndpoint;

    @Override
    public void onEnable() {
        // Plugin startup logic
        this.logger = getLogger();
        this.webSocketClientEndpoint = new WebSocketClientEndpoint(this);

        logger.info("Hello, World!");

        Objects.requireNonNull(getCommand("connect")).setExecutor(new ConnectCommandExecutor(this));
        Objects.requireNonNull(getCommand("disconnect")).setExecutor(new DisconnectCommandExecutor(this));
    }

    @Override
    public void onDisable() {
        // Plugin shutdown logic
        logger.info("Goodbye, World!");
    }
}

File: plugin/src/main/java/me/aurealisch/pudgeland/Pudgeland.kt
package me.aurealisch.pudgeland

import org.bukkit.plugin.java.JavaPlugin

class Pudgeland : JavaPlugin()
{
    override fun onEnable()
    {
        val drugs = mapOf<Drug, DrugItem>(
            Drug.SmokingPipe to DrugItem("Трубка для курения"),
            Drug.Syringe to DrugItem("Шприц"),
            Drug.TobaccoLeaf to DrugItem("Лист табака"),
            Drug.GooseberryLeaf to DrugItem("Лист крыжовника"),
            Drug.DriedTobaccoLeaf to DrugItem("Сушенный лист Табака"),
            Drug.DriedGooseberryLeaf to DrugItem("Сушенный лист Крыжовника"),
            Drug.DiamondCrystals to DrugItem("Кристаллы Алмаза"),
            Drug.SyringeWithCactus to DrugItem("Шприц с Кактусом"),
            Drug.RedGlowstonePowderSnuffMixture to DrugItem("Ред-глоустоуновая пороховая нюхательная смесь"),
            Drug.WeezerWheels to DrugItem("Визер колёса")
        )
    }

    override fun onDisable()
    {
        // Plugin shutdown logic
        logger.info("Goodbye, World!")
    }
}
